packages/blockchain-wallet-v4-frontend/src/modals/BuySell/EnterAmount/Checkout/template.success.tsx

```diff
@@ -450,6 +450,7 @@ const Success: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
               maxFontSize='56px'
               placeholder='0'
               autoComplete='off'
+              pointerToLeft={true}
```

Comment:
just add `pointerToLeft`. no need to pass `true`

Reply:
Got it!